pkg/distro/packagesets/loader.go

```diff
@@ -16,9 +16,9 @@ import (
 var Data embed.FS
 
 type packageSet struct {
-	Include   []string   `yaml:"include"`
-	Exclude   []string   `yaml:"exclude"`
-	Condition conditions `yaml:"condition,omitempty"`
+	Include   []string    `yaml:"include"`
+	Exclude   []string    `yaml:"exclude"`
+	Condition *conditions `yaml:"condition,omitempty"`
```

Comment:
Nitpick: this change is in commit `172e42b984047a8f8ec2ad40560bc7887c0e1079`, but based on the commit message, it does not belong here.